modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -121,6 +121,8 @@
 				[@subpath, @path, foo]
 				[, res://test.gd, bar]
 				[/codeblock]
+				[b]Note:[/b] This function is not recursive, which means that nested objects will not be represented as dictionaries. Also, properties of reference types ([Object], [Dictionary], [Array], and packed arrays) are copied by reference, not duplicated.
```

Comment:
I do still think that saying "properties passed by reference" is clearer and more familiar than "of reference types", especially because there's little to no common spot where this distinction is made clear once and for all.
I'm not dying on that hill, though.